examples/private_connection/main.tf

```diff
@@ -0,0 +1,190 @@
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "subnet" {
+  name                     = "forseti-subnet"
+  region                   = var.region
+  ip_cidr_range            = "10.127.0.0/20"
+  network                  = google_compute_network.network.self_link
+  private_ip_google_access = true
+  log_config {
+    aggregation_interval = "INTERVAL_10_MIN"
+    flow_sampling        = 1
+    metadata             = "INCLUDE_ALL_METADATA"
+  }
+}
+
+# Router to control egress only to Google APIs
+resource "google_compute_router" "forseti-router" {
+  name    = "forseti-network-router"
+  network = google_compute_network.network.name
+  region  = var.region
+}
+
+resource "google_compute_router_nat" "forseti-nat" {
+  name   = "forseti-network-router-nat"
+  router = google_compute_router.forseti-router.name
+  region = google_compute_router.forseti-router.region
+
+  nat_ip_allocate_option             = "AUTO_ONLY"
+  source_subnetwork_ip_ranges_to_nat = "ALL_SUBNETWORKS_ALL_IP_RANGES"
+
+  nat_ips                          = []
+  min_ports_per_vm                 = "64"
+  udp_idle_timeout_sec             = "30"
+  icmp_idle_timeout_sec            = "30"
+  tcp_established_idle_timeout_sec = "1200"
+  tcp_transitory_idle_timeout_sec  = "30"
+
+  log_config {
+    enable = true
+    filter = "ERRORS_ONLY"
+  }
+}
+
+# Private DNS zone to forward Google API calls
+resource "google_dns_managed_zone" "forseti-private-zone" {
+  name        = "forseti-private-zone"
+  dns_name    = "googleapis.com."
+  description = "private zone for Google API private access"
+  visibility  = "private"
+  private_visibility_config {
+    networks {
+      network_url = google_compute_network.network.self_link
+    }
+  }
+}
+
+resource "google_dns_record_set" "forseti-private-zone-cname" {
+  managed_zone = google_dns_managed_zone.forseti-private-zone.name
+  name         = "*.${google_dns_managed_zone.forseti-private-zone.dns_name}"
+  type         = "CNAME"
+  ttl          = 300
+  rrdatas      = ["private.googleapis.com."]
+}
+
+resource "google_dns_record_set" "forseti-private-zone-a" {
+  managed_zone = google_dns_managed_zone.forseti-private-zone.name
+  name         = "private.${google_dns_managed_zone.forseti-private-zone.dns_name}"
+  type         = "A"
+  ttl          = 300
+  rrdatas      = ["199.36.153.8", "199.36.153.10", "199.36.153.9", "199.36.153.11"]
+}
+
+# Route calls to private API via default-internet-router
+resource "google_compute_route" "forseti-private-vip-route" {
+  name             = "forseti-private-vip-route"
+  dest_range       = "199.36.153.8/30"
+  priority         = 100
+  network          = google_compute_network.network.name
+  next_hop_gateway = "default-internet-gateway"
+}
+
+# Forseti deployment.
+module "forseti" {
+  source               = "../../"
+  domain               = var.domain
+  project_id           = var.project_id
+  org_id               = var.org_id
+  manage_rules_enabled = false
```

Comment:
No need to include `manage_rules_enabled`

Reply:
done